v2/internal/testcommon/kube_per_test_context.go

```diff
@@ -246,19 +247,19 @@ func (tc *KubePerTestContext) Subtest(t *testing.T) *KubePerTestContext {
 	return result
 }
 
-// DefaultTimeoutReplaying is the default timeout for a single operation when replaying.
-var DefaultTimeoutReplaying = 2 * time.Minute
+// OperationTimeoutReplaying is the default timeout for a single operation when replaying.
+var OperationTimeoutReplaying = 30 * time.Second
```

Comment:
We might need to leave this at like 1m (I'm not sure how I picked 2m). If it passes at 30s though then awesome let's use that.

Reply:
If it takes 30s to achieve the desired state I think something has definitely gone wrong?

Reply:
```suggestion
var OperationTimeoutReplaying = 2 * time.Minute
```

Reply:
Restored to 2 minutes because apparently the Mongo/SQL ones do take a while.